README.md

```diff
@@ -6,7 +6,8 @@ Throttling middleware for [Telegraf](https://github.com/telegraf/telegraf) bot f
 
 ## Installation
 ```
-yarn install telegraf-throttler
+npm install telegraf-throttler
```

Comment:
Make it clearer that the only one of the two commands should be used

Reply:
Done.